client/lib/media/test/validation-store.js

```diff
@@ -340,5 +353,30 @@ describe( 'MediaValidationStore', function() {
 				101: [ MediaValidationErrors.FILE_TYPE_UNSUPPORTED ]
 			} );
 		} );
+
+		it( 'should detect an external media error and set error on site 0', () => {
```

Comment:
mmm, let me check that I follow what we are doing here: my understanding is that the MediaValidationStore keeps a list of errors for site/item; we use the 0 index to signal that the error isn't in a particular item but in a service. Is that alright? If that's the case, should we rename this test to `should detect an external media error and set an error on item 0`. And maybe instead of 0, we could use the already existing `ERROR_SITE_ID` constant in `client/lib/media/validation-store.js`.